src/main/java/fxlauncher/CreateManifest.java

```diff
@@ -55,6 +56,10 @@ public static void main(String[] args) throws IOException, URISyntaxException {
                 if (named.containsKey("stop-on-update-errors"))
                     stopOnUpdateErrors = Boolean.valueOf(named.get("stop-on-update-errors"));
 
+                // Configure stopOnUpdateErrors
```

Comment:
Can you fix this comment so it mentions updateText instead?

Reply:
I didn't realize when I was pasting :)